lib/pavilion/commands/group.py

```diff
@@ -122,14 +122,14 @@ def run(self, pav_cfg, args):
 
         return self._run_sub_command(pav_cfg, args)
 
-    def _get_group(self, pav_cfg, group_name: str) -> TestGroup:
+    def _get_group(self, pav_cfg: config.PavConfig, group_name: str, show_tracebacks: bool = False) -> TestGroup:
         """Get the requested group, and print a standard error message on failure."""
```

Comment:
I think in general with the group operations it's safe to ignore the 'show_tracebacks' option.

Reply:
Or even to let the first one get re-raised...